server/models/User.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require("mongoose");
+
+const UserSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    required: true,
+    unique: true
+  },
+  password: {
+    type: String,
+    required: true
+  }
+});
+
+module.exports = User = mongoose.model("user", UserSchema);
```

Comment:
i think you should separate this and capitalize user 
so its: 
let User = mongoose.model("User", UserSchema)
module.exports = User